src/diffusers/loaders.py

```diff
@@ -24,19 +25,23 @@
 import safetensors
 import torch
 from huggingface_hub import hf_hub_download, model_info
+from packaging import version
 from torch import nn
 
 from .models.modeling_utils import _LOW_CPU_MEM_USAGE_DEFAULT, load_model_dict_into_meta
 from .utils import (
     DIFFUSERS_CACHE,
     HF_HUB_OFFLINE,
     _get_model_file,
+    convert_diffusers_state_dict_to_peft,
+    convert_old_state_dict_to_peft,
```

Comment:
What's the difference between these two? Could we be a bit more explicit with the naming so that the distinction being made is very clear?

Reply:
Please check the refactor in [f8e87f6](https://github.com/huggingface/diffusers/pull/5058/commits/f8e87f6220a3ae2c63119e4f83f23f8cc321cbce) and let's move the discussion here: https://github.com/huggingface/diffusers/pull/5058#discussion_r1329894605